js/user-form.js

```diff
@@ -82,15 +89,32 @@ errors.forEach((value, key) =>
   )
 );
 
-imgUploadForm.addEventListener('submit', (evt) => {
-  evt.preventDefault();
-  pristine.validate();
-});
+const blockSubmitButton = () => {
+  submitButton.disabled = true;
+  submitButton.textContent = SubmitButtonText.SENDING;
+};
+
+const unblockSubmitButton = () => {
+  submitButton.disabled = false;
+  submitButton.textContent = SubmitButtonText.IDLE;
+};
+
+const setOnFormSubmit = (cb) => {
+  imgUploadForm.addEventListener('submit', async (evt) => {
```

Comment:
здесь тоже лучше через handler называть
```suggestion
const setFormSubmitHandler = (cb) => {
  imgUploadForm.addEventListener('submit', async (evt) => {
```

Reply:
Done.